### PR TITLE
When updating msw, also create mock service worker

### PR DESCRIPTION
**Pull request by jinmayamashita**
## Why

When the [msw version is updated](https://github.com/monstar-lab-oss/reactjs-boilerplate/pull/64), the service worker file is not updated and the test is broken.
e.g. https://github.com/monstar-lab-oss/reactjs-boilerplate/pull/84

## How

Added Github action to commit service worker file creation when `renovate/msw-**` PR is created.

### CODE REVIEW

**Comment by jinmayamashita** [comment]
Doesn't seem possible to test before merging?

File: .github/workflows/msw_init.yml [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+  pull_request_target:
     branches:
-      - "renovate/msw-**"
+      - main
     types: [opened]
 
 jobs:
-  build:
+  create_msw_file:
+    if: startsWith(github.head_ref, 'renovate/msw-')
```
Comment (jinmayamashita) [inline .github/workflows/msw_init.yml:11]: This will make an additional commit with a branch that starts with `renovate/msw-`

**Comment by jinmayamashita** [comment]
@G-N555 We've talked about this issue before so would love for you to review it for us, what do you think?

**Comment by G-N555** [comment]
> We've talked about this issue before so would love for you to review it for us, what do you think?

I agree, it would be good for us to updating msw to avoid conflict between service worker.
Thank you for your help for implementing this.🙇

**Review by G-N555 — APPROVED** [review]

**Pull request merged**